src/collectors/uniswapx_order_collector.rs

```diff
@@ -103,7 +106,7 @@ impl UniswapXOrderCollector {
 impl Collector<UniswapXOrder> for UniswapXOrderCollector {
     async fn get_event_stream(&self) -> Result<CollectorStream<'_, UniswapXOrder>> {
         let url = format!(
-            "{}/orders?orderStatus=open&chainId={}&orderType={}",
+            "{}/orders?orderStatus=open&chainId={}&orderType={}&limit=50",
```

Comment:
Ik we need this for V3 to work, but this would also be used for other orders. Just want to double-check if that's ok and if we should increase it above 50

Reply:
Yeah, good Q. The default is set to 50 on the OrderService if not specified so it should be fine.

Reply:
thinking about this more, we might run into scenario where there are more than 50 new orders at a given time, especially given the fact that order tracking is known to be flaky (there are already a few filled/expired priority orders still marked as open already apparently)

Reply:
bumped up to 500